e2e_test/batch/order/test_order_by_materialized_view.slt.part

```diff
@@ -0,0 +1,79 @@
+# Copyright 2022 Singularity Data
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# description: Test ordering a materialized view with underlying ordering
```

Comment:
@st1page As no other .slt.part file contained any license header except the one from DuckDB, I took the license from the source files. Is this okay?

Reply:
I think we do not need to do this. :thinking:

Reply:
Alright, I removed it